src/feed-list.scss

```diff
@@ -0,0 +1,150 @@
+@import "./mixins";
+
+$block: #{$fd-namespace}-feed-list;
+
+.#{$block} {
+  $fd-feed-list-padding-start: 1rem;
+  $fd-feed-list-padding-end: 0.5rem;
+  $fd-feed-list-margin-start: 0.75rem;
+  $fd-feed-list-margin-end: 0;
+  $fd-feed-list-padding-group: 0.75rem;
+  $fd-feed-list-padding-action-small: 0.75rem;
+  $fd-feed-list-icon-color: var(--sapContent_IconColor);
+
+  @include fd-reset();
+  @include fd-flex();
+
+  background-color: var(--sapField_Background);
+  max-width: 40rem;
+
+  .#{$block}__thumb {
+    @include fd-reset();
+
+    display: inline-flex;
+    margin-left: $fd-feed-list-margin-end;
+    margin-right: $fd-feed-list-margin-start;
+
+    @include fd-rtl() {
+      margin-left: $fd-feed-list-margin-start;
+      margin-right: $fd-feed-list-margin-end;
+    }
+  }
+
+  &__body {
+    @include fd-reset();
+
+    display: flex;
+    padding: 1rem;
+
+    & + .#{$block}__body {
+      border-top: var(--sapList_BorderWidth) solid var(--sapList_BorderColor);
+    }
+  }
+
+  &--no-border {
+    .#{$block}__body {
+      padding: 1rem 1rem 0.75rem;
+      border-top: none;
+
+      & + .#{$block}__body {
+        padding-top: $fd-feed-list-padding-group;
+        padding-bottom: $fd-feed-list-padding-group;
+      }
+
+      &:last-child {
+        padding-bottom: 1rem;
+      }
+    }
+  }
+
+  &__link {
+    &--more {
+      text-transform: uppercase;
+    }
+  }
+
+  &__actions {
+    @include fd-reset();
+
+    float: right;
+    margin: 0.5rem;
+    position: relative;
+    top: -1rem;
+    right: -1rem;
+
+    @include fd-rtl() {
+      float: left;
+      right: 1rem;
+      left: -1rem;
+    }

```

Comment:
Can you create variable for this `-1rem`?

Reply:
Sure will add